frontend/include/chpl/parsing/parser-error-classes-list.h

```diff
@@ -69,4 +69,5 @@ POSTPARSE_ERROR_CLASS(MultipleManagementStrategies, const uast::New::Management,
 POSTPARSE_ERROR_CLASS(PostParseErr, std::string)
 POSTPARSE_WARNING_CLASS(PostParseWarn, std::string)
 ERROR_CLASS(UnsupportedAsIdent, const uast::As*, const uast::AstNode*)
+WARNING_CLASS(ParenlessAttributeArgDeprecated, const uast::Attribute*)
```

Comment:
We try to keep the error classes lists sorted, both in the header and source files.